test/dag-node-test.spec.js

```diff
@@ -535,13 +550,4 @@ describe('DAGNode', () => {
     const deserializedObject = dagPB.util.deserialize(serializedObject)
     expect(deserialized.toJSON()).to.deep.equal(deserializedObject.toJSON())
   })
-
-  it('creates links from objects with .Size properties', () => {
-    const node = new DAGNode(uint8ArrayFromString('some data'), [{
-      Hash: 'QmUxD5gZfKzm8UN4WaguAMAZjw2TzZ2ZUmcqm2qXPtais7',
-      Size: 9001
-    }])
-
-    expect(node.Links[0].Tsize).to.eql(9001)
-  })
```

Comment:
Shouldn't this still work (I guess with `Tsize` instead of `Size`).

Reply:
It's more that `Hash` should be a CID but we do still handle the case.  Will reinstate.